test/src/test/java/org/corfudb/runtime/object/transactions/StreamTest.java

```diff
@@ -41,6 +44,67 @@
     private final int READ_PERCENT = 80;
     private final int MAX_PERCENT = 100;
 
+
+    @Test
+    public void sequencerTrimTest() {
+
+        SMRMap<String, String> map = instantiateCorfuObject(SMRMap.class, "A");
+        final int numEntries = 10;
+        TXBegin();
+        map.get("a");
+        t2(() -> {
+            TXBegin();
+            for (int x = 0; x < numEntries; x++) {
+                map.put(Integer.toString(x), Integer.toString(x));
+            }
+            TXEnd();
+        });
+
+        getRuntime().getAddressSpaceView().prefixTrim(1);
+
+        for (int x = 0; x < numEntries; x++) {
+            map.put(Integer.toString(x), Integer.toString(x));
+        }
+
+        boolean abortException = false;
+
+        try {
+            TXEnd();
```

Comment:
Just an alternate way of putting this:
```
try {
    TXEnd();
    fail();
} catch()...
```
You can avoid the flag and the assertions.